tools/swift-reflection-dump/swift-reflection-dump.cpp

```diff
@@ -137,11 +138,12 @@ class Image {
     for (SectionRef S : O->sections()) {
       if (!needToRelocate(S))
         continue;
-      StringRef Content;
-      if (auto EC = S.getContents(Content))
-        reportError(EC);
-      std::memcpy(&Memory[getSectionAddress(S)], Content.data(),
-                  Content.size());
+      if (llvm::Expected<llvm::StringRef> Content = S.getContents()) {
```

Comment:
`reportError` is noreturn.  You can verify that the value is invalid and if so, report the error.  That will avoid unneeded indentation.